config/test.js

```diff
@@ -61,4 +61,45 @@ module.exports = {
       "Zzn584h/xbSIcP9/4SXmEK3wtoueUtMkvS3yKTvk+w==\n" +
       "-----END RSA PRIVATE KEY-----",
   },
+
+  botToken: {
+    publicKey:
```

Comment:
are these keys different or the same as the above, if they are the same should we keep them here?

Reply:
They are same and we don't need distinct ones for development and testing purposes. We can keep them here as our production keys will be different from these, also we have done the same for `userToken` as well